tests/subsys/modem/modem_cmux/src/main.c

```diff
@@ -793,14 +793,14 @@ ZTEST(modem_cmux, test_modem_cmux_prevent_work_while_released)
 	/* Validate no new requests can be submitted */
 	modem_cmux_connect(&cmux);
 	modem_cmux_disconnect(&cmux);
-	modem_pipe_open(dlci1_pipe);
-	modem_pipe_open(dlci2_pipe);
+	modem_pipe_open(dlci1_pipe, K_SECONDS(10));
+	modem_pipe_open(dlci2_pipe, K_SECONDS(10));
```

Comment:
If `K_SECONDS(10)` is some kind of default as you mentioned in the commit message, then that should arguable be a `#define`d value IMO. 

For the sake of fixing CI, it's not something we need to do in this PR